Framework/Geometry/inc/MantidGeometry/Crystal/UnitCell.h

```diff
@@ -20,26 +20,46 @@
 namespace Mantid {
 namespace Geometry {
 /** @class UnitCell UnitCell.h Geometry/Crystal/UnitCell.h
-  Class to implement unit cell of crystals.
-  It is based on code by Laurent Chapon. It does not contain information about
-  lattice orientation.
-  See documentation about UB matrix in the Mantid repository.\n
-  For documentation purposes, units for lengths are assumed to be \f$ \mbox{ \AA
-  } \f$, and for reciprocal lattice lengths
-  \f$ \mbox{ \AA }^{-1} \f$,  but can be anything, as long as used consistently.
-  Note that the convention used for
-  reciprocal lattice follows the one in International Tables for
-  Crystallography, meaning that for an orthogonal lattice
-  \f$ a^{*} = 1/a \f$ , not   \f$ a^{*} = 2 \pi /a \f$
-
-  References:
-    - International Tables for Crystallography (2006). Vol. B, ch. 1.1, pp. 2-9
-    - W. R. Busing and H. A. Levy, Angle calculations for 3- and 4-circle X-ray
-  and neutron diffractometers - Acta Cryst. (1967). 22, 457-464
-
-
-  @author Andrei Savici, SNS, ORNL
-  @date 2011-03-23
+ Class to implement unit cell of crystals.
+ It is based on code by Laurent Chapon. It does not contain information about
+ lattice orientation.
+ See documentation about UB matrix in the Mantid repository.\n
+ For documentation purposes, units for lengths are assumed to be \f$ \mbox{ \AA
+ } \f$, and for reciprocal lattice lengths
+ \f$ \mbox{ \AA }^{-1} \f$,  but can be anything, as long as used consistently.
+ Note that the convention used for
+ reciprocal lattice follows the one in International Tables for
+ Crystallography, meaning that for an orthogonal lattice
+ \f$ a^{*} = 1/a \f$ , not   \f$ a^{*} = 2 \pi /a \f$
+
+ References:
+ - International Tables for Crystallography (2006). Vol. B, ch. 1.1, pp. 2-9
+ - W. R. Busing and H. A. Levy, Angle calculations for 3- and 4-circle X-ray
+ and neutron diffractometers - Acta Cryst. (1967). 22, 457-464
+
+
+ @author Andrei Savici, SNS, ORNL
+ @date 2011-03-23
+ Copyright &copy; 2007-8 ISIS Rutherford Appleton Laboratory, NScD Oak Ridge
```

Comment:
You appear to have reverted the license style changes here, I suggest look at this file in `master` to make it consistent.